api/blocklistener_test.go

```diff
@@ -7,10 +7,11 @@
 package api
 
 import (
-	"errors"
 	"testing"
 	"time"
 
+	"github.com/pkg/errors"
+
```

Comment:
delete the empty line
"github.com/pkg/errors" should be grouped with other third party packages
same for the rest